library/alloc/src/collections/vec_deque/mod.rs

```diff
@@ -2547,7 +2547,7 @@ impl<T, A: Allocator> VecDeque<T, A> {
     /// ```
     /// use std::collections::VecDeque;
     ///
-    /// let deque: VecDeque<_> = vec![0, 1, 1, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55].into();
+    /// let deque = VecDeque::from([0, 1, 1, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55]);
```

Comment:
The docs recommend using `into` over `from`, so could we keep using `into` in these cases, but drop the `vec!`? (And replace the various iterator constructs with `into` rather than `from` too)